GB-858

osman opened the pull request:
query-time biases

Commit:
GB-858

osman commented on src/main/java/com/groupbyinc/api/Query.java:
```diff
@@ -1193,4 +1186,72 @@ public Query addQueryUrlParams(String key, String value) {
         return this;
     }
+
+    /**
```
@willwarren can you please review this section. it is the initial documentation for query-time biases

codeallthethingz commented on src/main/java/com/groupbyinc/api/Query.java:
```diff
@@ -1193,4 +1186,72 @@ public Query addQueryUrlParams(String key, String value) {
+     * <code>
+     * Add a biasing profile, which is defined at query time. This allows you to override or augment
+     * biasing profiles defined in Command Center. By default, biases defined here will augment the ones
+     * defined in Command Center.
```
Lets add: 

`Set override to true to only use the biasing profile defined here and ignore the ones created in command center.`

codeallthethingz commented:
@osman @jmbtutor Quick change - could I also see this running?

jmbtutor committed:
Change query-time bias option from override to augment

The rest of the query overrides, so query-time biases should also
override by default.
